readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
That's fantastic @philpl.

I think the guards are a really interesting idea. This is the problematic use case that I see:

```js
styled.div`
  ${ props => props.red && `color: red;` }
`
```

We really need it to be ``css`color: red;` `` for the ISTF parsing to work, and I'm _really hesitant_ to try to have the babel step inject that call. Maybe the guard could log a warning saying "Inline CSS snippet injected without `css` wrapper. This won't be autoprefixed and may break future optimisations. For more information, see <link to docs>"